genesis-beta-tester.php

```diff
@@ -1,80 +0,0 @@
-<?php
```

Comment:
Hey, @ethan-schaffer this file would need to stay since removing it could cause trouble when users update from previous versions.

My suggestion would be keeping this file with the plugin comment header + the `register_activation_hook` and `genesis_beta_tester_init`, move the class `Genesis_Beta_Tester` to a file properly labeled (as I see that have been already done) inside a new directory labeled `includes/`, and finally, `require_once` the new class file in this file.

Please let me know if you have any question and I'll be happy to help :)